docs/docs/Integrations/Composio/integrations-composio.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Integrate Composio with Langflow
+slug: /integrations-composio
+---
+
+Langflow integrates with [Composio](https://docs.composio.dev/introduction/intro/overview) as a toolset for your **Agent** component.
+
+Instead of juggling multiple integrations and components in your flow, connect the Composio component to an **Agent** component to use all of Composio's supported APIs and actions as **Tools** for your agent.
+
+## Prerequisites
+
+- [Composio API key created](https://app.composio.dev/)
+- [SerpApi API key created](https://serpapi.com/)
+- [OpenAI API key created](https://platform.openai.com/)
+
+## Create a Composio tool
+
+1. Navigate to the [Composio application](https://app.composio.dev/dashboard).
+In the **Integrations** tab, navigate to the tool you want integrate with Langflow.
+This example uses the SerpApi tool in Composio.
```

Comment:
Why is this example using SerpApi instead of any of the other available options?

Reply:
+1 I was about to suggest that too.

We have used Gmail integration to send emails. It might be a good fit too.

Reply:
Agreed, switched the doc over to Gmail tool